docs/installguide/install_all.rst

```diff
@@ -7,6 +7,13 @@ Windows
 Upgrading KA Lite over an existing installation is easy -- just run the installer and follow the prompts!
 You don't need to uninstall your old KA Lite installation first.
 
+Upgrade
+_______
+
+To upgrade KA Lite on Windows, simply download the latest installer and run it.
+You can follow the prompts to either keep your existing data or delete old data and start over.

```

Comment:
It's true for the Windows installer -- though now that I'm looking at it closer I see we already had a note about upgrading. Maybe verify what I wrote under the linux section?